provider/aws/templates/resource/postgres.tmpl

```diff
@@ -46,12 +46,12 @@
       },
       "EngineVersion": {
         "Description": "Version of Postgres",
-        "Default": "9.6",
+        "Default": "12",
         "Type": "String"
       },
       "Family": {
         "Type": "String",
-        "Default": "postgres9.6",
+        "Default": "postgres12",
         "Description": "Postgres version family"
       },
```

Comment:
AWS deprecated the Postgress 9.6, see [Deprecation of PostgreSQL 9.6](https://docs.aws.amazon.com/AmazonRDS/latest/PostgreSQLReleaseNotes/postgresql-versions.html#postgresql-versions-version96-deprecation-notice).

AWS is recommending upgrading Postgres to 12, so I changed the template to use this version.

> You should upgrade all your PostgreSQL 9.6 DB instances to PostgreSQL 12 or higher as soon as possible. We recommend that you first upgrade to minor version 9.6.20 or higher and then upgrade directly to PostgreSQL 12 rather than upgrading to an intermediate major version